lib/DistributionProvider.pm

```diff
@@ -39,9 +39,9 @@ sub provide {
     return Distribution::Sle::15sp2->new() if is_sle('>15');
     return Distribution::Sle::15sp0->new() if is_sle('=15');
     return Distribution::Sle::12->new() if is_sle('12+');
+    return Distribution::Opensuse::AgamaDevel->new() if is_leap('16.0+');
```

Comment:
actually you need to implement Sle::16.0:: package, I noticed that reading your pr title and commit that is incorrect, this is not related with booting but with "Adapt distri pattern for Lea 16.0"

Reply:
Updated the pr title and commit.
So you mean add another line as Sle::16.0::package to support Leap 16.0 on OSD?